pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/worker/WorkerConfig.java

```diff
@@ -538,6 +538,12 @@ public String getFunctionAssignmentTopic() {
     )
     private boolean exposeAdminClientEnabled = false;
 
+    @FieldContext(
+            category = CATEGORY_FUNCTIONS,
+            doc = "Enables message batching between functions"
+    )
+    private boolean isFunctionBatchingEnabledDefault = true;
```

Comment:
This seems like a one-off config as in the there are no other configs in worker configs that define defaults for certain function configs.  If we are going to let cluster admin set defaults for function configs at a cluster level, we should do it in a more comprehensive manner and not just have one-off configs like this.